packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationStep.scss

```diff
@@ -1,3 +1,9 @@
 .BasicInformationStep {
   background-color: white;
+
+  // Temp styles start
```

Comment:
I applied some temp styles for the container, we will have to update this when we connect this component to the create community flow